### ISSUE TITLE
Specifying Camera doesn't work

### ISSUE THREAD

**Issue by AnthonyTTaylor** [unifi-toolbox/unifi-protect-video-downloader#54]
Was trying to download all footage from one camera named "Garage"
When passing in the --cameras flag with "Garage" the script runs, prints out the list of cameras and then terminates.
No files downloaded. 

Listing "all" for camera tag starts the download. Confirmed files are available to download.

Protect Versions 1.18.1
Downloader Version 2.0.2

**Comment by danielfernau** [comment]
Hi there!

The `--cameras` option takes either the input `all` or a comma-separated list of camera IDs, but not names. In the camera list you should see both, the name as well as it's ID. 

Try using something like `--cameras="a1b2c3d4e5f6abcdef123456`, and replace `a1b2c3d4e5f6abcdef123456` with the ID of your `Garage` camera - that should work.

**Comment by AnthonyTTaylor** [comment]
Interesting, there should be a different flow around this, as you won't be able to get the ID's until first run, only to have to rerun the program.

**Comment by danielfernau** [comment]
Yeah, I agree. Could be easier. I might add something in one of the next versions that makes camera selection more straightforward. 

But: you can also always get the camera ID(s) from the Protect Web UI. Open it in a web browser and log in. On the left side, click on `Devices`. Then select one of your cameras from the list. Now the browser address bar looks something like this: `https://<ip-of-protect-server>/protect/devices/a1b2c3d4e5f6abcdef123456`. That last part there is the camera ID you're looking for. From there just copy+paste it to your command line.
